src/Schemas/Schemas.vcxitems

```diff
@@ -15,10 +15,11 @@
   </ItemGroup>
   <ItemGroup>
     <ClInclude Include="$(MSBuildThisFileDirectory)ManifestSchema.h" />
+    <ClInclude Include="$(MSBuildThisFileDirectory)PackagesSchema.h" />
     <ClInclude Include="$(MSBuildThisFileDirectory)resource.h" />
   </ItemGroup>
   <ItemGroup>
-    <ResourceCompile Include="$(MSBuildThisFileDirectory)ManifestSchema.rc" />
+    <ResourceCompile Include="$(MSBuildThisFileDirectory)Schemas.rc" />

```

Comment:
>Schemas.rc [](start = 57, length = 10)

Last time I chatted with John, we think we'll need a new vcxitems for import/export and other schemas that's specifically for AppInstallerCli.exe
We should leave the manifest one untouched because this is also used in wingetutil

Reply:
Restored ManifestSchema and added a separate vcxitems in latest commit.